overlord/configstate/configcore/runwithstate.go

```diff
@@ -40,6 +40,9 @@ func init() {
 	// capture cloud information
 	addWithStateHandler(nil, setCloudInfoWhenSeeding, nil)
 
+	// home directory configuration
+	addWithStateHandler(validateHomedirsConfiguration, handleHomedirsConfiguration, nil)
```

Comment:
not sure whether it matters given that is more an admin option but it seems we don't use the state to handle this? so it could be a FSOnly option instead?

Reply:
Indeed. I've now declared it as a FSOnly handler.